shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/BlockChain.hs

```diff
@@ -626,8 +626,8 @@ mkSeed NeutralNonce slot nonce =
 -- in case the reference value `1 / (1 - p)` is above the exponential function
 -- at `-σ * c`, `BELOW` if it is below or `MaxReached` if it couldn't
 -- conclusively compute this within the given iteration bounds.
-checkVRFValue :: Natural -> Rational -> ActiveSlotCoeff -> Bool
```

Comment:
`p` is the output of the VRF translated to unit interval so `p \in [0, 1)`.

```
-- then           p < 1 - (1 - f)^σ
-- <=>  1 / (1 - p) < exp(-σ * c)
```

The `LHS >= 1`, `1 <= RHS <= 1.1` (roughly, for `f = 0.1`).

Reply:
Note that the comment in the function is wrong, although the comparison is right.